fix-npm-build-for-vs2019.cmd

```diff
@@ -0,0 +1,2 @@
+npm config set msbuild_path "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild.exe" -g
```

Comment:
This file scares me a little. :-)

Reply:
node-sass forces an old version of node-gyp, which is broken if your machine only has vs 2019 on it.